src/dotnet-svcutil/lib/src/CodeDomFixup/CodeDomVisitors/NamespaceFixer.cs

```diff
@@ -21,7 +21,7 @@ internal class NamespaceFixup : CodeDomVisitor
         public NamespaceFixup()
         {
             // Microsoft.Xml.dll
-            var msxmlTypes = TypeLoader.LoadTypes(typeof(Microsoft.Xml.XmlDocument).GetTypeInfo().Assembly, Verbosity.Silent);
+            var msxmlTypes = TypeLoader.LoadTypes(typeof(Microsoft.Xml.XmlDocument).GetTypeInfo().Assembly, Verbosity.Silent, null, null);
```

Comment:
is this change necessary?

Reply:
Yes, I added two parameters in LoadTypes method